src/scss/3_components/form/throbber.scss

```diff
@@ -0,0 +1,115 @@
+/**
+ * @file
+ * Visual styles for Drupal's throbber when ajax calls are in progress.
+ *
+ * @see throbber.js and https://www.drupal.org/node/2974681
+ */
+@import '../../init';
+
+// Throbber
```

Comment:
coding standards might flag the inline commented code, so perhaps remove these?

Reply:
Yep will do